spec/integ/matrix-client-event-timeline.spec.ts

```diff
@@ -1692,27 +1652,11 @@ describe("MatrixClient event timelines", function () {
                         encodeURIComponent(THREAD_ROOT.event_id!) +
                         "/" +
                         encodeURIComponent(THREAD_RELATION_TYPE.name) +
-                        buildParams(Direction.Backward, "start_token"),
+                        buildRelationPaginationQuery({ dir: Direction.Backward, limit: 1 }),
                 )
                 .respond(200, function () {
                     return {
-                        original_event: THREAD_ROOT,
-                        chunk: [],
-                    };
-                });
-            httpBackend
-                .when(
-                    "GET",
-                    "/_matrix/client/v1/rooms/!foo%3Abar/relations/" +
-                        encodeURIComponent(THREAD_ROOT.event_id!) +
-                        "/" +
-                        encodeURIComponent(THREAD_RELATION_TYPE.name) +
-                        buildParams(Direction.Forward, "end_token"),
-                )
-                .respond(200, function () {
-                    return {
-                        original_event: THREAD_ROOT,
-                        chunk: [THREAD_REPLY],
+                        chunk: [THREAD_ROOT],
                     };
                 });
             const timeline = await flushHttp(client.getEventTimeline(timelineSet, THREAD_ROOT.event_id!));
```

Comment:
Where is the test coverage for the issues that this claims to fix?

 - https://github.com/vector-im/element-web/issues/23942
 - https://github.com/vector-im/element-web/issues/23916
 
I don't see it amongst these random edits here.